syntax/blocks.js

```diff
@@ -333,6 +331,50 @@ disambig("pen.setColor", "pen.setHue", function(children, lang) {
   return (last.isInput && last.isColor) || last.isBlock
 })
 
+disambig("microbit.whenGesture", "gdxfor.whenGesture", function(
```

Comment:
It would be great to add some unit tests for this, like the existing tests for disambiguation! But I'd understand if you don't feel like doing that :stuck_out_tongue:

Reply:
Gonna do it